### ISSUE TITLE
meijerg.is_number is wrong

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy#13071]
```py
>>> meijerg([[],[]], [[S(1)/2],[0]], 1).is_number
False
>>> meijerg([[],[]], [[S(1)/2],[0]], 1).evalf()
0.513016136561828
```

This is because Expr.is_number just checks the args, and Basic.is_number defaults to False, which is what is used for the Tuple args of meijerg. meijerg should probably implement its own is_number to work around this.

**Comment by vedantc98** [comment]
Can I take up this issue?

**Comment by gxyd** [comment]
Closing, fixed by #13558 .
